pkg/services/registry.go

```diff
@@ -95,6 +104,8 @@ type Resolver interface {
 	Resolve(ctx context.Context, did w3c.DID, opts *ResolverOpts) (IdentityState, error)
 	ResolveGist(ctx context.Context, opts *ResolverOpts) (*GistInfo, error)
 	BlockchainID() string
+	GetWalletAddress() (string, error)
+	TypedData(primaryType PrimaryType, did w3c.DID, identityState IdentityState, walletAddress string) (apitypes.TypedData, error)
```

Comment:
not sure that it must be interface method for resolver. Can it be private  if it is not called outside of resolver?

Reply:
Both GetWalletAddress() and TypedData() are being called in DidDocumentServices (pkg/services/did.go)